intranet/settings/__init__.py

```diff
@@ -452,7 +452,6 @@ def get_month_seconds():
     "DEFAULT_AUTHENTICATION_CLASSES":
     ("intranet.apps.api.authentication.ApiBasicAuthentication",
      "oauth2_provider.contrib.rest_framework.OAuth2Authentication"),
-    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticated",)
```

Comment:
That key was duplicated. It is also set at the top of the dictionary, and since the values are exactly the same I removed this one.

TL;DR: No functionality change.

Reply:
Ah, GitHub didn't show that in the diff.